packages/http-client-csharp/generator/Microsoft.Generator.CSharp.ClientModel/test/mocks/tspCodeModel.json

```diff
@@ -0,0 +1,2923 @@
+{
```

Comment:
For unit test we should probably just construct the Input types the way we want them vs reading from this file since this file has way more in it than is actually in scope for the tests.

Reply:
The reason I added this was because the `Configuration.Load` method will throw if it can't find either the config or tsp code models files.